sansio_lsp_client/client.py

```diff
@@ -190,7 +347,7 @@ def recv(self, data: bytes) -> t.List[Event]:
 
         events: t.List[Event] = []
         for message in messages:
-            if isinstance(message, Response):
+            if isinstance(message, Response) or isinstance(message, ResponseList):
                 events.append(self._handle_response(message))
```

Comment:
This doesn't seem correct, because `_handle_response` doesn't handle `ResponseList`.